doc/CMakeLists.txt

```diff
@@ -57,6 +57,9 @@ if (DOXYGEN_FOUND)
 
 	find_package(Perl)
 
+	add_executable (markdownlinkconverter markdownlinkconverter/markdownlinkconverter.c)
+	find_util (markdownlinkconverter MARKDOWN_LINK_CONVERTER "")
+
```

Comment:
> Ok, then it won't work with mingw

So if I would use the mingw gcc to generate the docu it will fail?

So I can leave the CmakeList.txt with the find _util or should I change it back?